app/views/prof_landing/index.html.erb

```diff
@@ -0,0 +1,114 @@
+<!-- app/views/prof_landing/landing.html.erb -->
+<html>
+  <head>
+    <title>Professor Landing Page</title>
+    <style>
+      /* Light Mode Styles */
+      body,
+      html {
+        margin: 0;
+        padding: 0;
+        overflow: hidden;
+      }
+      body {
+        background-color: #fff;
+        color: #333;
+      }
+      .form-control {
+        background-color: #f8f9fa; /* Light gray for input fields */
+        color: #333;
+      }
+      .btn.btn-danger.custom-maroon-button {
+        background-color: maroon;
+        border-color: maroon;
+        color: #fff;
+      }
+      a,
+      label {
+        color: #007bff; /* Blue for links and labels in light mode */
+      }
+
+      /* Dark Mode Styles */
+      html.dark-mode body {
+        background-color: #333;
+        color: #fff;
+      }
+      html.dark-mode .form-control {
+        background-color: #444;
+        color: #fff;
+      }
+      html.dark-mode .btn.btn-danger.custom-maroon-button {
+        background-color: maroon;
+        border-color: maroon;
+        color: #fff;
+      }
+      html.dark-mode a,
+      html.dark-mode label {
+        color: #bbb; /* Lighter color for links and labels in dark mode */
+      }
+
+      .vertical-center-container {
+        display: flex;
+        flex-direction: column;
+        align-items: center; /* Center horizontally */
+        justify-content: center; /* Center vertically */
+        height: 100vh; /* Make the container full height of the viewport */
+        overflow: hidden; /* Hide overflow content */
+      }
+
+      .vertical-center-container a {
+        margin-right: 0;
+        margin-bottom: 10px;
+      }
+
+      .btn-equal-size {
+        width: 150px;
+        height: 40px;
+        margin-bottom: 10px;
+      }
+    </style>
```

Comment:
We will need to move this into `app/assets/stylesheets`. That's the standard we should follow so that CSS can be maintained and reused all in one place.

Reply:
Yeah, that's the plan for next iteration.